### PR TITLE
	Fixed footers, recover-password page, add background-video

### PR DESCRIPTION
**Pull request by akhlopiachyi**

### CODE REVIEW

File: library/agency-about-2.html [hunk 1]
```diff
@@ -166,4 +166,5 @@
     </div>
   </div>
 </div>
-${require('./layouts/footer.html')}
```
Comment (mod) [inline library/agency-about-2.html:169]: are you sure agency has no footer ?

File: library/blog-post-alt.html [hunk 2]
```diff
@@ -17,10 +16,14 @@ <h3 class="blog-post-title">
       <div class="blog-post-share">
         Share this post: 
         <a href="#">
-          <i class="ion-social-twitter"></i>
-        </a>
+          <i class="ion-social-twitter">
+		  				
+	 				</i>
```
Comment (mod) [inline library/blog-post-alt.html:21]: fix properly icons

File: library/blog.html [hunk 3]
```diff
@@ -60,8 +59,7 @@
             <img src="/images/uifaces/9.jpg" class="avatar">
             By <span>Christopher Waltz</span>
           </div>
-          <a href="#">
-          <img class="bg" src="/images/unsplash/photo-1470753937643-efeb931202a9.jpg"/>
+          <a href="#" class="bg" style="background-image:url('images/unsplash/photo-1470753937643-efeb931202a9.jpg');">
```
Comment (mod) [inline library/blog.html:62]: put the correct require;

File: library/career-post.html [hunk 4]
```diff
@@ -127,4 +127,4 @@
 </div>
 
 
-${require('./layouts/footer.html')}
+${require('./layouts/footer-blog.html')}
```
Comment (mod) [inline library/career-post.html]: this page is not related to blog
I don't like the name of the footer.
maybe it's a footer-light ? or footer-simple ?
what is different with standard footer ?

File: library/index-dark.html [hunk 5]
```diff
@@ -1,5 +1,269 @@
+<!doctype html>
```
Comment (mod) [inline library/index-dark.html:1]: I don't understand why you don't use the dark-header and footer here

File: library/index-photography.html [hunk 6]
```diff
@@ -132,7 +132,7 @@
 <div class="index-photography-footer" id="contact">
   <div class="footer-wrapper skrollr" 
     data-200-top="transform: translateY(-120px);box-shadow: 0px 55px 60px rgba(0,0,0,0.3);" 
-    data-bottom="transform: translateY(-40px);box-shadow: 0px 0px 0px rgba(0,0,0,0.3);">
+    data-bottom="transform: translateY(-40px);box-shadow: 0px 0px 0px rgba(0,0,0,0.3);" style="transform: translateY(-120px); box-shadow: 0px 55px 60px rgba(0,0,0,0.3);">
```
Comment (mod) [inline library/index-photography.html:135]: rejected, we don't write style="" anymore

**Review by mod — CHANGES_REQUESTED** [review]
Most changes are not going in the right direction, instead of fixing the template with light fixes, we drifting heavily away

File: library/career-post.html (hunk 4 above)
Comment (ghost) [reply library/career-post.html]: standart footer was footer-big-menu, but we need to have footer-blog(i changed it on footer-dark)
